test/Recipe-test.js

```diff
@@ -125,7 +57,71 @@ describe('Recipe Class', function() {
       ]
     };
 
-    recipe = new Recipe(recipeDatum);
+    recipe = new Recipe(recipeDetails);
+  });
+
+  it('should have be a function', function() {
+    expect(Recipe).to.be.a('function');
   });
 
+  it('should be an instance of Recipe', function() {
+    expect(recipe).to.be.an.instanceOf(Recipe);
+  });
+
+  desbribe('Recipe constructor properties', function() {
+    it('should have an id', function() {
+      expect(recipe.id).to.equal(595736);
+    });
+
+    it('should store an image url', function() {
+      expect(recipe.image).to.equal("https://spoonacular.com/recipeImages/595736-556x370.jpg");
+    });
+
+    it('should be able to all ingredient\'s information', function() {
+      const ingredients = recipe.getIngredients(recipeDetails.ingredients);
+
+      expect(recipe.ingredients).to.be.an('array');
+      expect(recipe.ingredients.length).to.deep.equal(3);
+      expect(ingredients).to.deep.equal(recipe.ingredients);
```

Comment:
I may have the same question for lines 83-85 as I do for lines 91-93, 101-103, and line 111, but this one is a little different, so I'm not sure.